docs/src/main/paradox/release-notes/releases-1.1.md

```diff
@@ -0,0 +1,55 @@
+# Release Notes (1.1.x)
+
+## 1.1.0-M1
+
+Release notes for Apache Pekko 1.1.0-M1. See [GitHub Milestone](https://github.com/apache/incubator-pekko/milestone/2?closed=1) for fuller list of changes.
+
+### 1.0.x changes
+
+Apache Pekko 1.1.0-M1 contains all the changes that have been released in the @ref:[1.0.x releases](releases-1.0.md), with the exception of the changes to support interacting with Akka remote and cluster instances ([PR765](https://github.com/apache/incubator-pekko/pull/765), [PR1112](https://github.com/apache/incubator-pekko/pull/1112)).
+
+### Bug Fixes
+
+* in TlsActor, flush data to user only after handshake has finished ([PR1128](https://github.com/apache/incubator-pekko/pull/1128))
```

Comment:
https://github.com/apache/incubator-pekko/pull/1156

Reply:
adding 1156 - 1088 is listed as a fix in 1.0.x and this page starts by saying all 1.0.x changes are included in 1.1.0-M1